schort.py

```diff
@@ -5,23 +5,15 @@
 
 app = Flask(__name__)
 
-@app.route('/', methods=['GET', 'POST'])
-@app.route('/<shortLink>', methods=['GET', 'POST'])
+@app.route('/', methods=['GET', 'POST', 'HEAD'])
+@app.route('/<shortLink>', methods=['GET', 'POST', 'HEAD'])
 def short(shortLink=""):
 	if request.method == "GET":
```

Comment:
I'd prefer to extend this with ` or request.method == "HEAD"` and keep the rest of the logic equal to the one from GET requests.